2-Browsers/Week1/assignment/ex4-whatsTheTime/index.js

```diff
@@ -6,8 +6,24 @@ Full description at: https://github.com/HackYourFuture/Assignments/tree/main/2-B
   second). Use `setInterval()` to make sure the time stays current.
 2. Have the function execute when it's loading in the browser.
 ------------------------------------------------------------------------------*/
+
 function addCurrentTime() {
-  // TODO complete this function
+  const elem = document.createElement('h1');
+  elem.textContent = 'current time';
+  document.body.append(elem);
+
+  const paragraph = document.createElement('p');
+  document.body.append(paragraph);
+
+  function getCurrentTime() {
+    const time = new Date();
+    const hours = (time.getHours() + '').padStart(2, '0');
+    const min = (time.getMinutes() + '').padStart(2, '0');
+    const sec = (time.getSeconds() + '').padStart(2, '0');
+    paragraph.textContent = `hours: ${hours}  minutesgit: ${min} second : ${sec}`;
```

Comment:
There is a small typo here 😄 `minutesgit`

```suggestion
    paragraph.textContent = `hours: ${hours}  minutes: ${min} second : ${sec}`;
```